File: update_executator.py
class UpdateExecutator():
    def __init__(self, service):
        self.service = service
        self._id = None

    def configure(self, config: dict):
        self._id = config.get('id', None)

    def execute(self):
        events_service = self.service.events()

        id = self._id #'827gj35869mlimobovmajc1m94'
        event = events_service.get(calendarId='primary', eventId=id).execute()

        event['summary'] = 'Appointment at Somewhere - Update'

        updated_event = events_service.update(calendarId='primary', eventId=event['id'], body=event).execute()
        
        # Print the updated date.
        print(updated_event['updated'])

File: golang/update_executator.go
package main

import (
	"fmt"
	"log"

	"google.golang.org/api/calendar/v3"
)

type UpdateExecutator struct {
	service *calendar.Service
	id      string
}

func NewUpdateExecutator(service *calendar.Service) *UpdateExecutator {
	return &UpdateExecutator{
		service: service,
	}
}

func (ue *UpdateExecutator) Configure(config map[string]string) {
	ue.id = config["id"]
}

func (ue *UpdateExecutator) Execute() {
	eventService := ue.service.Events

	event, err := eventService.Get("primary", ue.id).Do()
	if err != nil {
		log.Fatalf("Failed to retrieve event: %v", err)
	}

	event.Summary = "Appointment at Somewhere - Update"

	updatedEvent, err := eventService.Update("primary", ue.id, event).Do()
	if err != nil {
		log.Fatalf("Failed to update event: %v", err)
	}

	fmt.Printf("Event updated: %s\n", updatedEvent.Updated)
}

func main() {
	updateExecutator := NewUpdateExecutator(service)

	updateExecutator.Configure(map[string]string{"id": "827gj35869mlimobovmajc1m94"})
	updateExecutator.Execute()
}
